packages/corvid-local-server/test/it/localDocumentChanges.spec.js

```diff
@@ -63,8 +66,11 @@ describe("local changes", () => {
           onDocumentChange
         );
 
-        let filePath = localSiteBuilder.getLocalFilePath(docItem);
-        let fileContent = localSiteBuilder.getLocalFileContent(sc[itemKey]());
+        const filePath = localSiteBuilder.getLocalFilePath(docItem, "page");
```

Comment:
asking for "page" is only correct if the itemKey is page/pageWithCode/lightbox/lightboxWithCode